BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -188,20 +188,18 @@ open class BraintreeClient @VisibleForTesting internal constructor(
     @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
     fun sendAnalyticsEvent(
             eventName: String,
-            payPalContextId: String? = null,
-            linkType: String? = null,
-            isVaultRequest: Boolean = false
+            params: AnalyticsEventParams = AnalyticsEventParams.Builder().build()
```

Comment:
If we were to remove the builder, we could have params nullable here and default to null.

Reply:
Addressed: [930bef6](https://github.com/braintree/braintree_android/pull/1031/commits/930bef6fd4fef8eb137784283e87d9ac294b6b3b)